app.rb

```diff
@@ -0,0 +1,55 @@
+require 'sinatra/base'
+require 'sinatra/activerecord'
+require './config/environment'
+
+# Global for verbose output
+# $verbose = true
+
+class Peep < ActiveRecord::Base 
```

Comment:
I really like the use of the ActiveRecord class, simplifies the whole CRUD process.